### PR TITLE
Do not import dumps and loads from galaxy.util.json .

### PR DESCRIPTION
**Pull request by nsoranzo**
Also fix some typos.

### CODE REVIEW

File: lib/galaxy/webapps/galaxy/controllers/dataset.py [hunk 1]
```diff
@@ -21,8 +21,6 @@
 
 comptypes = []
 
-# TODO: not used in this file
-from galaxy.util.json import loads  # noqa
```
Comment (jmchilton) [inline lib/galaxy/webapps/galaxy/controllers/dataset.py:25]: Any clue why this was here?

Comment (nsoranzo) [inline lib/galaxy/webapps/galaxy/controllers/dataset.py:25]: Commit 6135cb5764597aa9459f3fed16fddd8701c4807f by @bgruening.

Comment (dannon) [inline lib/galaxy/webapps/galaxy/controllers/dataset.py:25]: It was only moved in that commit, not added.

Comment (nsoranzo) [inline lib/galaxy/webapps/galaxy/controllers/dataset.py:25]: Yep, but it should have been removed (it was a flake8 commit).

**Comment by dannon** [comment]
+1

**Pull request merged**